java/daikon/chicory/Instrument.java

```diff
@@ -611,11 +848,11 @@ private static boolean isVoid(Type t) {
   }
 
   /**
-   * Transforms return instructions to first assign the result to a local variable
-   * (return__$trace2_val) and then do the return. Also, calls Runtime.exit() immediately before the
-   * return.
+   * If this is a return instruction, generate new il to assign the result to a local variable
+   * (return__$trace2_val) and then call Runtime.exit(). This il wil be inserted immediately before
```

Comment:
This is confusing.  It's not `java.lang.Runtime.exit()`, nor `daikon.Runtime`.  I think it's what is in `runtime_classname`.  Clarifying that would be helpful.